x-pack/dockerlogbeat/docs/configuration.asciidoc

```diff
@@ -0,0 +1,358 @@
+tries to reconnect. If the attempt fails, the backoff timer is increased
+exponentially up to `backoff.max`. After a successful connection, the backoff
+timer is reset.
+|1s
+
+|`output.logstash.backoff.max`
+|The maximum number of seconds to wait before attempting to connect to
+{ls} after a network error.
+|60s
+
+|`output.logstash.bulk_max_size`
+|The maximum number of events to bulk in a single {ls} request. Specify 0 to
+allow the queue to determine the batch size. 
+|2048
+
+|`output.logstash.compression_level`
+|The gzip compression level. Valid compression levels range from 1 (best speed)
+to 9 (best compression). Specify 0 to disable compression.  Higher compression
+levels reduce network usage, but increase CPU usage.
+|0
+
+|`output.logstash.escape_html`
+|Whether to escape HTML in strings.
+|`false`
+
+|`output.logstash.loadbalance`
+|Whether to load balance when sending events to multiple {ls} hosts. If set to
+`false`, the driver sends all events to only one host (determined at random) and
+switches to another host if the selected one becomes unresponsive.
+|`false`
+
+|`output.logstash.pipelining`
+|The number of batches to send asynchronously to {ls} while waiting for an ACK
+from {ls}. Specify 0 to disable pipelining.
+|2
+
+|`output.logstash.proxy_url`
+|The URL of the SOCKS5 proxy to use when connecting to the {ls} servers. The
+value must be a URL with a scheme of `socks5://``. You can embed a
+username and password in the URL (for example,
+`socks5://user:password@socks5-proxy:2233`).
+|
+
+|`output.logstash.proxy_use_local_resolver`
+|Whether to resolve {ls} hostnames locally when using a proxy. If `false`,
+name resolution occurs on the proxy server.
+|`false`
+
+|`output.logstash.slow_start`
+|When enabled, only a subset of events in a batch are transferred per
+transaction. If there are no errors, the number of events per transaction
+is increased up to the bulk max size (see `output.logstash.bulk_max_size`).
+On error, the number of events per transaction is reduced again.
+|`false`
+
+|`output.logstash.timeout`
+|The number of seconds to wait for responses from the {ls} server before
+timing out. 
+|30
+
+|`output.logstash.ttl`
+|Time to live for a connection to {ls} after which the connection will be
+re-established. Useful when {ls} hosts represent load balancers. Because
+connections to {ls} hosts are sticky, operating behind load balancers can lead
+to uneven load distribution across instances. Specify a TTL on the connection
+to distribute connections across instances. Specify 0 to disable this feature.
+This option is not supported if `output.logstash.pipelining` is set.
+|0
+
+|`output.logstash.worker`
+|The number of workers per configured host publishing events to {ls}. Use with
+load balancing mode (`output.logstash.loadbalance`) set to `true`. Example:
+If you have 2 hosts and 3 workers, in total 6 workers are started (3 for each
+host).
+|1
+
+|=====
+
+
+[float]
+=== Logging options
```

Comment:
Not sure if we want to add this, since extracting logs from the docker plugin is a tad convoluted, particularly on osx.

Reply:
OK I removed this section